operators/cmd/tenant-operator/main.go

```diff
@@ -88,6 +91,9 @@ func main() {
 	flag.StringVar(&ncURL, "nc-url", "", "The base URL for the nextcloud actor.")
 	flag.StringVar(&ncTnOpUser, "nc-tenant-operator-user", "", "The username of the acting account for nextcloud.")
 	flag.StringVar(&ncTnOpPsw, "nc-tenant-operator-psw", "", "The password of the acting account for nextcloud.")
+	flag.DurationVar(&requeueTimeMinimum, "nc-tenant-operator-rq-time-min", 4*3600*1000000000, "Minimum nanoseconds before requeue of controller.") // 30000000000 for testing
```

Comment:
```suggestion
	flag.DurationVar(&requeueTimeMinimum, "nc-tenant-operator-rq-time-min", 4*time.Hour, "Minimum time elapsed before requeue")
```

Use the appropriate constants, instead of vanilla numbers. Same below.

Reply:
Resolved in [12a16cd](https://github.com/netgroup-polito/CrownLabs/pull/821/commits/12a16cd6a828cbfe11f1e9646627a51c8986bef1) for all 3 lines.